File: AppDiDong-master/AppDiDong-master/app_travel/lib/data/model/flightbooking_dto.dart
class FlightBookingDTO {
  final int userId;
  final int? flightId;
  final DateTime? bookingDate;
  final String? status;
  final String? flightName;
  final String? flightImageUrl;

  FlightBookingDTO({
    required this.userId,
    this.flightId,
    this.bookingDate,
    this.status,
    this.flightName,
    this.flightImageUrl,
  });

  factory FlightBookingDTO.fromJson(Map<String, dynamic> json) {
    return FlightBookingDTO(
      userId: json['userId'],
      flightId: json['flightId'],
      bookingDate: json['bookingDate'] != null ? DateTime.parse(json['bookingDate']) : null,
      status: json['status'],
      flightName: json['flightName'],
      flightImageUrl: json['flightImageUrl'],
    );
  }
}

File: TravelAPI/TravelAPI/TravelAPI/DTOs/FlightBookingDTO.cs
namespace TravelAPI.DTOs
{
    public class FlightBookingDTO
    {
        public int UserId { get; set; }
        public int FlightId { get; set; }
        public DateTime BookingDate { get; set; }
        public string Status { get; set; }
        public string FlightName { get; set; }
        public string FlightImageUrl { get; set; }
        public string Location { get; set; }
        public string Destination { get; set; }
        public double TotalPrice { get; set; }
        public DateTime DepartureTime { get; set; }
        public int DayFlight { get; set; }
        public int NumberOfTickets { get; set; }
    }
}
